bindings/README.md

```diff
@@ -0,0 +1,19 @@
+# Matrix Rust SDK bindings
+
+In this directory, one can find bindings to the Rust SDK that are
+maintained by the owners of the Matrix Rust SDK project.
+
+* [`apple`] or `matrix-rust-components-swift`, Swift bindings of the [`matrix-sdk`] crate,
```

Comment:
We should probably note that these are generated from `matrix-sdk-ffi` (or remove them ^^)